plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -846,10 +846,12 @@ class FossId internal constructor(
     ): ScanResult {
         // TODO: Maybe get issues from FossID (see has_failed_scan_files, get_failed_files and maybe get_scan_log).
 
-        // TODO: Deprecation: Remove the pending files in issues. This is a breaking change.
-        val issues = rawResults.listPendingFiles.mapTo(mutableListOf()) {
-            Issue(source = name, message = "Pending identification for '$it'.", severity = Severity.HINT)
-        }
+        val issues = mutableListOf<Issue>()
```

Comment:
Commit message question:
> However, counting the unique pending files having snippets is not an option since some pending files could have no snippet at all.

If you have the list of pending files and the snippet findings including their text locations a few lines below, can't you filter the pending files based on that? I'm not saying it should be done, I just wonder if I understand the commit message correctly.

Reply:
Current state:
The list of file pending identifications are present as issues in the scan reults. Users use this data to find out if any file requires identification in FossID.

New states:
There is now the snippets in the scans results. One pending file can have one or more snippets.
However a file pending identification does not have necessary snippets. Or all its snippets have a match type ignored by ORT.

Therefore, for the user to find out if a file requires identification, collecting the pending files associated to the snippets is not enough. Hence, this commit add an issue containing the pending file count.
This could have been done before the introduction of the snippets model, but at the time we were unsure if the snippet model could help here.